### PR TITLE
keep tar and gzip tools installed in the final image

### PR DESCRIPTION
**Pull request by ddsharpe**
To keep Image Tool images in line with Java server JRE images, this change will leave the TAR and GZIP tools installed in the final image.  Previously, these tools were installed in an intermediate layer, and then left out of the final image.

### CODE REVIEW

File: imagetool/src/main/resources/docker-files/Create_Image.mustache [hunk 1]
```diff
@@ -187,12 +188,6 @@ ENV ORACLE_HOME={{{oracle_home}}} \
     PROPERTIES_FILE_DIR={{{oracle_home}}}/properties \
     PATH=${PATH}:{{{java_home}}}/bin:{{{oracle_home}}}/oracle_common/common/bin:{{{oracle_home}}}/wlserver/common/bin:{{{oracle_home}}}{{#isWdtEnabled}}:${{{domain_home}}}/bin{{/isWdtEnabled}}
 
-## Create user and group
```
Comment (rjeberhard) [inline imagetool/src/main/resources/docker-files/Create_Image.mustache]: This change seems unrelated to tar/gzip.

Comment (ddsharpe) [reply imagetool/src/main/resources/docker-files/Create_Image.mustache]: If you mean the create user and group, then yes, its related.  The OS_UPDATE intermediate layer that was originally thrown away already has the user and group created.  So, there is no reason to create them again.
